tap_aggregator/src/server.rs

```diff
@@ -456,33 +464,36 @@ mod tests {
             .build(format!("http://127.0.0.1:{}", local_addr.port()))
             .unwrap();
 
-        // Create 100 receipts
+        // Create 300 receipts
         let mut receipts = Vec::new();
-        for _ in 1..100 {
+        for _ in 1..300 {
             receipts.push(
-                EIP712SignedMessage::new(Receipt::new(allocation_ids[0], 42).unwrap(), &keys.0)
-                    .await
-                    .unwrap(),
+                EIP712SignedMessage::new(
+                    Receipt::new(allocation_ids[0], u128::MAX / 1000).unwrap(),
+                    &keys.0,
+                )
+                .await
+                .unwrap(),
             );
         }
 
         // Skipping receipts validation in this test, aggregate_receipts assumes receipts are valid.
         // Create RAV through the JSON-RPC server.
-        // Test with only 10 receipts
+        // Test with only 250 receipts
```

Comment:
Leaving this comment to be a little more vague will mean it doesn't need to be updated every time (or risk going out of sync).
```suggestion
        // Test with a number of receipts that stays within request size limit
```